src/server/controllers/match.controller.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const matchSchema = require('mongoose').model('match');
+import mailer from 'nodemailer';
+
+/*
+ * Methods about mentoring request, accept or reject including E-mail Service
+ */
+
+let matchData = new matchSchema();
+
+let sampleSuccessResult = {
+  successCode: 1,
+  matchData: matchData,
```

Comment:
what's the difference between the first matchData and the second? If they're different variables we should name them differently.